include/galsim/fmath/fmath.hpp

```diff
@@ -849,3 +854,26 @@ __m128 (*const exp_ps)(__m128) = local::C<>::getInstance().exp_ps_;
 inline float exp2(float x) { return fmath::exp(x * 0.6931472f); }
 
 } // fmath
+
+#else // __x86_64__
+
+namespace fmath {
+
+inline float exp(float x)
+{
+  return expf(x);
+}
+
+inline double expd(double x)
+{
+  return exp(x);
```

Comment:
I think I'd prefer `std::exp` here.  Otherwise, I'm not 100% confident that the resolution order won't favor the above float function and lose precision.